OneBlink.SDK/FormsClient.cs

```diff
@@ -307,4 +317,9 @@ public async Task<SubmissionDataUrl> GenerateSubmissionDataUrl(long formId, stri
             return url;
         }
     }
+
+    public class WebhookRequest
+    {
+        public string callbackUrl { get; set; }
+    }

```

Comment:
While we understand why you have added this class to use as a POST request payload, it does not belong in this file. If you would like it in this code base, move it to its own file under the `models` directory.

Reply:
This was accidental, We also need to make this request, but I believe we will be doing this outside of the SDK. Reverting.